bdd/step-definitions/e2e/host-steps.ts

```diff
@@ -28,10 +28,15 @@ let actualHealthResponse: any;
 let actualStatusResponse: any;
 let actualApiResponse: Response;
 let sequence: SequenceClient;
+let sequence1: SequenceClient;
```

Comment:
@a-tylenda could you perhaps refactor this a bit. These could be pushed to separate files as this one is growing quite a lot.

Reply:
fixed in https://github.com/scramjetorg/transform-hub/pull/82